### ISSUE TITLE
non-existing version in install_requires for pafy in 4239f7b187a49c450c7cbe20b326e285f8d2c0a2

### ISSUE THREAD

**Issue by glitsj16** [mps-youtube/mps-youtube#396]
Commit 4239f7b187a49c450c7cbe20b326e285f8d2c0a2 seems to refer to a non-existing pafy version 0.3.82:

    install_requires=['pafy >= 0.3.82, != 0.4.0, != 0.4.1, != 0.4.2']

https://github.com/mps-youtube/pafy/releases/latest has 0.3.80 as latest release.

After changing setup.py to reflect that it all works again:

    install_requires=['pafy >= 0.3.80, != 0.4.0, != 0.4.1, != 0.4.2']

**Comment by ids1024** [comment]
Yes, mpsyt git requires pafy git. Try searching for and opening a playlist, and you'll find it crashes.

**Comment by glitsj16** [comment]
Confusing.. I just did another test. Compiled both python-pafy & mps-youtube from git.

$ mpsyt
...
pkg_resources.DistributionNotFound: The 'pafy!=0.4.0,!=0.4.1,!=0.4.2,>=0.3.82' distribution was not found and is required by mps-youtube

Initially I installed pafy 0.3.80 via pip in an attempt to get these pafy dependencies fixed. I also re-installed my distribution's python-pafy-git (0.4.2.r19.g6b98e9b-1from AUR in arch linux).. This made me assume that 0.3.80 was needed. My bad. I should have taken a closer look at mpsyt --version, which mentions the pafy in use (amongst other relevant info). Rectified that and tried again, this time making sure there's only the python-pafy-git installed.

In that scenario I keep getting the same error:
    pkg_resources.DistributionNotFound: The 'pafy!=0.4.0,!=0.4.1,!=0.4.2,>=0.3.82'
    distribution was not found and is required by mps-youtube

Exactly like the original error I saw that triggered this issue report. So all in all I still need to edit the install_requires to get a working mps-youtube. Curent working edit:
    install_requires=['pafy >= 0.4.2, != 0.4.0, != 0.4.1']

$ mpsyt --version
mpsyt version  : 0.2.6-dev 
   notes       : development version
pafy version   : 0.4.2
Python version : 3.5.1 (default, Dec  7 2015, 12:58:09)

What am I missing here? How (where) does one get pafy version 0.3.82?
